server/app/routers/tools.py

```
@@ -98,16 +100,18 @@ def create_tool(tool: schemas.ToolCreate, db: Session = Depends(get_db)):
 
     except Exception as e:
         import traceback
+
         print("❌ Error during tool submission:", e)
         traceback.print_exc()
         raise HTTPException(status_code=500, detail="Internal Server Error")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)